.github/workflows/pythonpackage.yml

```
@@ -38,6 +38,7 @@ jobs:
         rm -r docs
         cp -r ../python-client/docs/_build/html ./docs
         git --no-pager diff --stat
+        if [ -z "$(git --no-pager diff --stat -- . ':(exclude)docs/_sources/index.rst.txt')" ]; then git checkout .; fi
         if (git --no-pager diff --stat | grep -q "2 files changed, 2 insertions(+), 2 deletions(-)"); then exit 1; fi
```

How about this?

```suggestion
        if [ -z "$(git --no-pager diff --stat -- . ':(exclude)docs/_sources/index.rst.txt')" ]; then exit 1; fi
        if (git --no-pager diff --stat | grep -q "1 files changed, 1 insertions(+), 1 deletions(-)"); then exit 1; fi
```

1. `git checkout .` is to make the next line failed, right? If so, `exit 1` is better from maintenance point of view.
2. `"2 files changed, 2 insertions(+), 2 deletions(-)"` seems not working
   * Before there was one more line changed at `docs/.buildinfo`, but there is no changes at `docs/.buildinfo` anymore
      * https://github.com/ki4070ma/python-client-sphinx-1/pull/2/files#

---

I could not remember the reason of `2 files changed, 2 insertions(+), 2 deletions(-)" `, but if the line did the same thing, to find the diff only in `docs/_sources/index.rst.txt`, then yes. It simply can fail.

> Before there was one more line changed at docs/.buildinfo, but there is no changes at docs/.buildinfo anymore. (It isn't so robust as well, though)

Ok, then, do you think we can remove `if (git --no-pager diff --stat | grep -q "2 files changed, 2 insertions(+), 2 deletions(-)"); then exit 1; fi`?

---

Sorry, it's my mistake. (My suggestion was wrong.)
Let me clarify it again.

We'd like to skip below case.
So in my understanding, just `if (git --no-pager diff --stat | grep -q "docs/_sources/index.rst.txt | 2 +-"); then exit 1; fi` is enough.


```
$git --no-pager diff --stat                                              
 docs/_sources/index.rst.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
```


`[ -z "$(git --no-pager diff --stat -- . ':(exclude)docs/_sources/index.rst.txt')" ]` will miss the case which `index.rst.txt` has more changes.